server/datastore/mysql/schema.sql

```diff
@@ -193,6 +193,7 @@ CREATE TABLE `host_users` (
   `created_at` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
   `removed_at` timestamp NULL DEFAULT NULL,
   `user_type` varchar(255) DEFAULT NULL,
+  `shell` varchar(255) DEFAULT NULL,
```

Comment:
Given that this can be null. While it probably won't be, ever, if it ever is, the select below will fail because `Shell` is `string` and not `*string`. You can either make Shell a pointer and make sure to check for nils when needed.

Or you can make this default to an empty string, and leave the struct the way it is.